diff --git a/cnn.py b/cnn.py
index 58baeee..9191110 100644
--- a/cnn.py
+++ b/cnn.py
@@ -93,7 +93,7 @@ def trainModel(train_data_generator, val_data_generator, model, initial_epoch):
 
 
 def _main():
-
+    sess = tf.Session(config=tf.ConfigProto(log_device_placement=True))
     # Create the experiment rootdir if not already there
     if not os.path.exists(FLAGS.experiment_rootdir):
         os.makedirs(FLAGS.experiment_rootdir)
